tests/phpunit/tests/user/query.php

```diff
@@ -162,13 +162,18 @@ public function test_get_all_primed_users() {
 		$filter = new MockAction();
 		add_filter( 'update_user_metadata_cache', array( $filter, 'filter' ), 10, 2 );
 
-		new WP_User_Query(
+		$query = new WP_User_Query(
 			array(
 				'include' => self::$author_ids,
 				'fields'  => 'all',
 			)
 		);
 
+		$users = $query->get_results();
+		foreach ( $users as $user ) {
+			$user->roles;
```

Comment:
I understand this is just to initiate loading the role data, but it looks like an error:

<img width="474" height="154" alt="Image" src="https://github.com/user-attachments/assets/884ac531-e15e-4542-88af-cff2eadf3b24" />

So I suggest adding an assertion to at least look at the value:

```suggestion
			$this->assertIsArray( $user->roles );
			foreach ( $user->roles as $role ) {
				$this->assertIsString( $role );
			}
```